manager/integration/tests/test_migration.py

```diff
@@ -220,7 +223,70 @@ def test_migration_with_rebuilding_replica(clients, volume_name):  # NOQA
     7. Observe volume migrated to node 2 (single active engine).
     8. Validate initially written test data.
     """
-    return
+    # Step 1
+    client = get_random_client(clients)
+    current_host = common.get_self_host_id()
+    host1, host2 = get_hosts_for_migration_test(clients) # NOQA
```

Comment:
Why keep adding `# NOQA`? any purpose?

Reply:
Sorry, I removed the unnecessary comment